packages/elements/src/card/__test__/card.test.js

```diff
@@ -82,6 +82,36 @@ describe('card/Card', () => {
       await oneEvent(el, 'item-trigger');
       expect(menu.opened).to.equal(false, 'Menu should close when item is selected');
     });
+    it('Should open menu and not close when clicked on non value item', async () => {
```

Comment:
Let's avoid arrow functions as described in the [doc](https://mochajs.org/#arrow-functions). Note that, I don't think we should refactor everything. Just make sure that we don't use arrow functions on the new test cases.